### DISCUSSION TITLE
Add Footer

### DISCUSSION THREAD

**Discussion by Khushboo288** [BBaoVanC/bobatheme#30] [Ideas]
This page needs a footer. I can add it to the page. Plz assign me this issue.

**Comment by Khushboo288** [comment]
Hey @BBaoVanC I m hacktoberfest contributer, so assign this to me only if this project is a part of Hactoberfest 2021. 
Also add label to this issue.

**Comment by BBaoVanC** [comment]
> This page needs a footer. I can add it to the page. Plz assign me this issue.

My original style had a footer, but I don't think it quite fits with the current design.

**Comment by BBaoVanC** [comment]
What did you have in mind to put on the footer?

**Comment by Khushboo288** [comment]
I can add favicon , back to top buttons, copywrite line, and footer navigation butttons. You choose what features you want , I will add that. Plz add hacktoberfest 2021 label to this issue. Look at the below pic for reference.
assign this to me only if your project is genuinely a part of hacktoberfest 2021.
![Screenshot (151)](https://user-images.githubusercontent.com/84862960/136750183-7e157d47-bc46-4ba4-96cc-08a380f31506.png)
